### ISSUE TITLE
[Accessibility][JAWS][Chapter/Subchapter Performance Report]: Screen reader is skipping the edit/collapsed input field, even though it is visibly present on the page.

### ISSUE THREAD

**Issue by nikitaorliak-cengage** [cengage/react-magma#2201]
[A11Y-4178](https://jira.cengage.com/browse/A11Y-4178)

Environment:
Windows 11/Chrome 137/JAWS 2025
Windows 11/Firefox 147/NVDA 2025

Steps to reproduce:
1. Open the URL – https://react-magma.cengage.com/version/4.10.0/api/combobox/
2. Turn on the screen reader (JAWS/NVDA/VoiceOver)
3. Go to the `Basic Usage` section
4. Move focus to the Combobox
5. Observe that the label "Hello" is not announced by the screenreader

--------------------------------------------------------------------------------------------

In the [A11Y-4178](https://jira.cengage.com/browse/A11Y-4178) they mentioned that it's also not announced on other platforms like Chrome after these steps:

5. Select any item from the combobox
6. Navigate to the whole page with the **down arrow key**
7. Observe that the selected field is not announced  

--------------------------------------------------------------------------------------------

Actual Result:
The screen reader does not announce or detect the Placeholder/Selected fields, even though it is visible to sighted users.

Expected Result:
The Placeholder/Selected fields should be fully accessible and announced by the screen reader, including both the label "Combobox" and the selected value.

WCAG Guideline:
1.3.1: Info and Relationships

User Impact:
Users relying on screen readers will not know what chapter is selected or shown. This creates confusion and lack of context, especially in forms or editable sections where chapter identification is necessary.

Recommendation:
Ensure the Selected field is not hidden using display: none, visibility: hidden, or aria-hidden="true". If it is dynamically inserted, verify that it is added to the DOM in a way that is accessible.

Screencast:
https://app.screencast.com/AVyZKffJAjbrV

Our screencast:
<img width="680" height="172" alt="Image" src="https://github.com/user-attachments/assets/87670594-a49f-40c3-8d34-b37fb42591f7" />

**Comment by olehnoskov-cengage** [comment]
**Tested on:**
- V5: https://storybook-preview-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/combobox--default
- V4: https://storybook-preview-v4-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/combobox--default

Everything works as expected. No bugs found.
